README.md

```diff
@@ -40,103 +83,65 @@ To run a specific unittest:
+And edit docker-compose.yml to change the FAKE_DNS environment variable to match. This will cause Boulder to use the local system resolver available on your host if one is available.
 
 Alternatively, you can override the docker-compose.yml default with an environmental variable using -e (replace 172.17.0.1 with the host IPv4 address found in the command above)
 
     docker-compose run -e FAKE_DNS=172.17.0.1 --service-ports boulder ./start.py
 
-Boulder's default VA configuration (`test/config/va.json`) is configured to
-connect to port 5002 to validate HTTP-01 challenges and port 5001 to validate
-TLS-SNI-01 challenges. If you want to solve challenges with a client running on
-your host you should make sure it uses these ports to respond to validation
-requests, or update the VA configuration's `portConfig` to use ports 80 and 443
-to match how the VA operates in production and staging environments. If you use
-a host-based firewall (e.g. `ufw` or `iptables`) make sure you allow connections
-from the Docker instance to your host on the required ports.
+Boulder's default VA configuration (`test/config/va.json`) is configured to connect to port 5002 to validate HTTP-01 challenges and port 5001 to validate TLS-SNI-01 challenges. If you want to solve challenges with a client running on your host you should make sure it uses these ports to respond to validation requests, or update the VA configuration's `portConfig` to use ports 80 and 443 to match how the VA operates in production and staging environments. If you use a host-based firewall (e.g. `ufw` or `iptables`) make sure you allow connections from the Docker instance to your host on the required ports.
 
-If a base image changes (i.e. `letsencrypt/boulder-tools`) you will need to rebuild
-images for both the boulder and bhsm containers and re-create them. The quickest way
-to do this is with this command:
+If a base image changes (i.e. `letsencrypt/boulder-tools`) you will need to rebuild images for both the boulder and bhsm containers and re-create them. The quickest way to do this is with this command:
 
     ./docker-rebuild.sh
 
-Slow start
-----------
+### Without Docker
 
-If you can't use the Docker setup, here are instructions for setting up a
-Boulder development environment without it.
+If you can't use the Docker setup, here are instructions for setting up a Boulder development environment without it.
 
-We recommend setting git's [fsckObjects
-setting](https://groups.google.com/forum/#!topic/binary-transparency/f-BI4o8HZW0/discussion)
-for better integrity guarantees when getting updates.
-
-Boulder requires an installation of libtool-ltdl, goose, SoftHSM, and MariaDB 10.1 to work correctly. If you want to save some trouble installing MariaDB and SoftHSM you can run them using Docker:
+Boulder requires an installation of libtool-ltdl, goose, SoftHSMv2, and MariaDB 10.1 to work correctly. If you want to save some trouble installing MariaDB and SoftHSM you can run them using Docker:
```

Comment:
Now that this section is called "Without Docker," we should probably remove the advice to run MariaDB and SoftHSMv2 via Docker.